### DISCUSSION TITLE
Distribution of releases

### DISCUSSION THREAD

**Discussion by samuelsayag** [helmfile/helmfile#119] [Questions] [answered]
Hello,

Would it be possible to have the helmfile binaries distributed like in the old repository and not just having sources?

LIke this:

![image](https://user-images.githubusercontent.com/6291619/170856978-9da84409-0178-4cbd-a7d1-1e0521d385a5.png)

Thanks!

**Comment by yxxhero** [comment]
It will be published in the near future.

**Comment by yxxhero** [comment, marked as answer]
done
